skorch/tests/callbacks/test_training.py

```diff
@@ -105,6 +115,22 @@ def epoch_3_scorer(net, *_):
         assert net.history[2, 'event_cp'] is True
         assert all((x is False) for x in net.history[3:, 'event_cp'])
 
+    def test_save_targets(
+            self, save_params_mock, save_history_mock, pickle_dump_mock,
+            net_cls, checkpoint_cls, data):
+        sink = Mock()
```

Comment:
You are assigning a sink but don't really use it anywhere. Why not stay with the default?

Reply:
Just a copy-paste left-over